proteus/NumericalSolution.py

```diff
@@ -1761,7 +1761,8 @@ def calculateSolution(self,runName):
                             self.systemStepController.sequenceStepTaken(model)
                     #end model split operator step
                     if stepFailed:
-                        systemStepFailed = not self.systemStepController.retrySequence_modelStepFailure()
+#                        systemStepFailed = not self.systemStepController.retrySequence_modelStepFailure()
```

Comment:
I think you should be able to get this behavior with  `maxSolverFailures=1` in your _n file.